traveler-swift-core/TravelerKit/Models/Details/Historic/PurchasedBookingItemDetails.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  PurchasedBookingItemDetails.swift
+//  TravelerKit
+//
+//  Created by Rakin Hoque on 2019-12-03.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+// TODO: Make decodable once backend changes to support single model aggregation
+/// The historic information of a `BookingItem` combined with `PurchasedBookingProduct`
+public struct PurchasedBookingItemDetails: CatalogItemDetails, PurchasedProduct {
```

Comment:
I dont think the details should be considered a purchased product...a PurchasedBookingProduct (or item or whatever) is a PurchasedProduct with (IMO)....actually now that I think of the naming thing I propose the following: 
BookingItem: Product, PurchasedBookingItem: PurchasedProduct. I'm not sure of the attribute differences between Product and Purchased Product, but I think that then it flows nicely with our naming and our existing modeling. Thoughts?

Reply:
Added TODO to remove the PurchasedProduct conformance

Reply:
Renamed to PurchasedBookingProductDetails and added TODO for renaming "Booking" to "Experience" in BookingItem.swift